### ISSUE TITLE
Version bump to get latest code live?

### ISSUE THREAD

**Issue by vmpj** [oxdi/vagrant-nixos#5]
Can we do a version bump to 0.0.7 to get the latest code out there?

**Comment by ctheune** [comment]
Poke. Here, too. :) :+1:

**Comment by ctheune** [comment]
This plugin is becoming non-functional as the network interface configuration has changed and now fails completely with additional private networks and static addresses.

**Comment by nha** [comment]
Is this plugin still functional and/or maintained ? If not, what would it take to make it work ?
Thanks :)

**Comment by ctheune** [comment]
I have not heard from oxdi at all. I would be happy to fork this by now and make releases und a new name.

**Comment by nha** [comment]
That's the power of open-source :) Using this with another box image (https://atlas.hashicorp.com/larryweya/boxes/nixos-14.12_64), nixOS boots, then using the external configuration file without options, files are generated in /etc/nixos/.. But you are right I did not manage to get the network working again. That would be really great if you forked it :)

**Comment by nha** [comment]
Does the last push from @chrisfarms  (https://github.com/oxdi/vagrant-nixos/commit/d7a0a983b30c473738f5abfca6d64fe9f502c545) change anything ?

I don't remember seeing it here. The date seems wrong though so either my memory is failing, or the push date differs from the commit date ?

**Comment by ctheune** [comment]
That is the one that I also have in my fork. :(

**Comment by ctheune** [comment]
I tried contacting Chris directly right now. Lets see whether he's still around.

**Comment by nha** [comment]
@ctheune Ah my memory is already failing... Yes that would be great :)

**Comment by Tzrlk** [comment]
Any luck?

**Comment by ctheune** [comment]
Nope. We started forking this now. See https://github.com/zimbatm/vagrant-nixos-plugin and https://rubygems.org/gems/vagrant-nixos-plugin
